contracts/scripts/foundry/DeployL2L1BlocksContract.s.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.10;
+
+import {Script} from "forge-std/Script.sol";
+import {console} from "forge-std/console.sol";
+
+import {L1Blocks} from "../../src/L2/L1Blocks.sol";
+
+contract DeployL2L1BlocksContract is Script {
```

Comment:
Since we need a hardfork to add this change, I think it's better to make it a predployed contract.